msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -23,41 +25,49 @@ Teams supports access to apps during meeting for the following meeting types:
 * [**One-on-one calls**](https://support.microsoft.com/office/start-a-call-from-a-chat-in-teams-f5138c9d-df4c-43d8-9cf6-53400c1a7798): Calls initiated in one-on-one chat.
 * [**Group calls**](https://support.microsoft.com/office/start-a-call-from-a-chat-in-teams-f5138c9d-df4c-43d8-9cf6-53400c1a7798): Calls initiated in group chat.
 * [**Instant meetings**](https://support.microsoft.com/office/start-an-instant-meeting-in-teams-ff95e53f-8231-4739-87fa-00b9723f4ef5): Meetings initiated through **Meet now** button in Teams calendar.
+* [**Webinar**](https://support.microsoft.com/office/get-started-with-teams-webinars-42f3f874-22dc-4289-b53f-bbc1a69013e3): Webinar initiated through **Webinar** button under **New Meeting** dropdown.
 
-Users can add apps to the meeting using the **+** option from their Teams meeting window.
+> [!NOTE]
+>
+> * Apps for instant meetings, scheduled public channel meetings, one-on-one, and group calls are currently available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * Currently app experiences are not supported in Teams instant channel meetings.
 
-:::image type="content" source="../assets/images/apps-in-meetings/add-app.png" alt-text="Add an app in meeting" border="true":::
+Learn more about [Teams meetings, expiration and policies](/MicrosoftTeams/meeting-expiration) and [Meetings, webinars, and live events](/microsoftteams/quick-start-meetings-live-events).
 
-Visit the [Teams store](https://go.microsoft.com/fwlink/p/?LinkID=2183121) and explore apps designed specifically for meetings.
+## Meeting lifecycle
 
-> [!Note]
->
-> * Currently when a third person is added to a one-on-one call, the call is elevated to a group call that means a new session starts. Apps added to the one-on-one call are not available in the group call. However, they can be added again.
->
-> * Currently app experiences are not supported in Teams instant channel meetings.
+Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the attendee's status.
```

Comment:
Is attendee's **status** the same as user type here (federated, guest, etc.) or is it related to attendee vs presenter vs organizer?

If dependent on meeting role:
```suggestion
The meeting lifecycle includes pre-meeting, in-meeting, and post-meeting scenarios. These experiences may vary depending on a user's role in the meeting.
```

If dependent on user type:
```suggestion
The meeting lifecycle includes pre-meeting, in-meeting, and post-meeting scenarios. These experiences may vary depending on the type of user.
```

Suggest to link to user types and/or meeting roles above on "**role in the meeting**" or "**type of user**". There could be confusion here on the connection between attendee status and user type.